README.md

```diff
@@ -204,6 +235,36 @@ To run all examples and play around with the code you have to run two separate c
 - `yarn dev` - runs webpack and node dev server
 - `yarn build:dev` - runs rollup in watch mode - livereload if you do changes to the insights library
 
+## Node.js module
+
+Insights library can be used on backend as a Node.js module.
+
+```js
+const aa = require("search-insights);
+aa("init", {
+  appId: "APPLICATION_ID",
+  apiKey: "SEARCH_API_KEY"
+});
+```
+
+It exports the same `aa` interface.
+
+### setUserToken
+
+On node environment, unlike browser environment, `setUserToken` must be called before sending any event.
```

Comment:
```suggestion
On the node environment, unlike the browser environment, `setUserToken` must be called before sending any event.
```
maybe it's just my frenglish that makes me add `the` everywhere ? 🤓 
maybe @Haroenv can confirm?

Reply:
maybe it's just my konglish not using `the` enough.

Reply:
your suggestions seem better @tkrugg